### PR TITLE
Customizer: ES6ify Controller and Route Defs

### PR DESCRIPTION
**Pull request by ockham**
Needed for https://github.com/Automattic/wp-calypso/pull/12998.

To test: Make sure that the customizer still works.

### CODE REVIEW

**Comment by ockham** [comment]
Failing CircleCI test is ESLint complaining about `sites-list` :man_facepalming:  WELL THAT'S WHY WE'RE DOING THIS!

File: client/my-sites/customize/controller.js [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-					query: context.query,
-					panel: context.params.panel
-				} )
-			),
-			document.getElementById( 'primary' )
-		);
-	}
-
-};
+	ReactDom.render(
```
Comment (ehg) [inline client/my-sites/customize/controller.js]: Drive-by, but what about using one of the helpers, which may be more conducive to single-tree-codemodding?

Comment (ockham) [inline client/my-sites/customize/controller.js]: Ugh, I'm getting a bit bogged down in dependency chains I'm building up for myself already. #12998 will take care of this, tho.
